prime-angular-frontend/src/app/modules/gis-enrolment/pages/ldap-information-page/ldap-information-page.component.html

```diff
@@ -39,12 +39,26 @@
           </mat-form-field>
 
         </div>
-        <div class="col-12 text-right">
-          <button mat-button
-                  type="button"
-                  color="primary">Forgot your password?
-          </button>
-        </div>
+
+        <ng-container *ngIf="hasAttemptedSubmission && remainingAttempts <= 1 && lockoutTimeInHours">
+          <app-alert type="danger"
+                     icon="error_outline">
+            <ng-container #alertContent
+                          *ngIf="remainingAttempts === 1"
+                          class="alert-content">
+              You have had multiple failed login attempts the next failed attempt will lock your account, and you
+              will need to wait {{ lockoutTimeInHours }} hour(s) before trying again. If you need assistance please
+              contact Reg Admin at (999) 999-9999.
+            </ng-container>
+
+            <ng-container #alertContent
+                          *ngIf="!remainingAttempts"
+                          class="alert-content">
+              Your account is now locked due to too many failed login attempts. Please wait {{ lockoutTimeInHours }}
+              hour(s) before trying again. If you think this is a mistake please contact Reg Admin at (999) 999-9999.
+            </ng-container>
```

Comment:
wonder if below sample would reduce the redundancy a little bit.
```html
<ng-container #alertContent class="alert-content">
  <ng-container *ngIf="remainingAttempts === 1; else #lockedBlock">
     You have had multiple failed login attempts the next failed attempt will lock your account, and you
     will need to wait {{ lockoutTimeInHours }} hour(s) before trying again. If you need assistance please
     contact Reg Admin at (999) 999-9999.
  </ng-container>
  <ng-template #lockedBlock>
     Your account is now locked due to too many failed login attempts. Please wait {{ lockoutTimeInHours }}
     hour(s) before trying again. If you think this is a mistake please contact Reg Admin at (999) 999-9999.
  </ng-template>
<ng-container>
```

Reply:
I had something similar to Johnny's suggestion in that example branch did that not work?

Reply:
The first `*ngIf` it should filter out anything that is greater than 1

```
<ng-container *ngIf="hasAttemptedSubmission && remainingAttempts <= 1 && lockoutTimeInHours">
```